docs/user/alerting/troubleshooting/alerting-common-issues.asciidoc

```diff
@@ -0,0 +1,227 @@
+[role="xpack"]
+[[alerting-common-issues]]
+=== Common Issues
+
+This page describes how to resolve common problems you might encounter with Alerting.
+
+[float]
+[[rules-small-check-interval-run-late]]
+==== Rules with small check intervals run late
+
+*Problem*
+
+Rules with a small check interval, such as every two seconds, run later than scheduled.
+
+*Solution*
+
+Rules run as background tasks at a cadence defined by their *check interval*.
+When a Rule *check interval* is smaller than the Task Manager <<task-manager-settings,`poll_interval`>>, the rule will run late.
+
+Either tweak the <<task-manager-settings,{kib} Task Manager settings>> or increase the *check interval* of the rules in question.
+
+For more details, see <<task-manager-health-scheduled-tasks-small-schedule-interval-run-late>>.
+
+
+[float]
+[[scheduled-rules-run-late]]
+==== Rules run late
+
+*Problem*
+
+Scheduled rules run at an inconsistent cadence, often running late.
+
+Actions run long after the status of a rule changes, sending a notification of the change too late.
+
+*Solution*
+
+Rules and actions run as background tasks by each {kib} instance at a default rate of ten tasks every three seconds.
+When diagnosing issues related to Alerting, focus on the tasks that begin with `alerting:` and `actions:`.
+
+Alerting tasks always begin with `alerting:`. For example, the `alerting:.index-threshold` tasks back the <<rule-type-index-threshold, index threshold stack rule>>.
+Action tasks always begin with `actions:`. For example, the `actions:.index` tasks back the <<index-action-type, index action>>.
+
+For more details on monitoring and diagnosing task execution in Task Manager, see <<task-manager-health-monitoring>>.
+
+[float]
+[[connector-tls-settings]]
+==== Connectors have TLS errors when executing actions
+
+*Problem*
+
+When executing actions, a connector gets a TLS socket error when connecting to
+the server.
+
+*Solution*
+
+Configuration options are available to specialize connections to TLS servers,
+including ignoring server certificate validation, and providing certificate
+authority data to verify servers using custom certificates.  For more details, 
+see <<action-settings,Action settings>>.
+
+[float]
+[[rules-long-execution-time]]
+==== Identify long-running rules
```

Comment:
Should we move this to the event log page since it's basically just a big event log query?

Reply:
The original idea was to keep all "common issues", like current (an issue with the long running rules) on the same page, where user can browse throw the list and find something similar to his problem. Does it make sense?